.gitlab/functional_test/system_probe.yml

```diff
@@ -54,6 +54,8 @@ pull_test_dockers_arm64:
     when: always
     paths:
       - $DD_AGENT_TESTING_DIR/kitchen-junit-*.tar.gz
+      - $CI_PROJECT_DIR/kitchen_logs
```

Comment:
Idea for a later improvement (for easier future maintenance): I wonder if we could reuse the artifacts content from `.kitchen_common` using references, with something like (not tested, would need experimentation):
```
artifacts:
    expire_in: 2 weeks
    when: always
    paths:
      - $DD_AGENT_TESTING_DIR/kitchen-junit-*.tar.gz
      - !reference [.kitchen_common, artifacts, paths]
```
That way, if the common list of artifacts changes, its effects are also applied to the system-probe kitchen tests.

Reply:
After some testing, that is not supported yet by Gitlab 😢: https://gitlab.com/gitlab-org/gitlab/-/issues/351169

Reply:
Got it, will undo that change.